pkg/controlplane/apiserver/options/options.go

```diff
@@ -191,6 +195,9 @@ func (s *Options) AddFlags(fss *cliflag.NamedFlagSets) {
 
 	fs.StringVar(&s.ServiceAccountSigningKeyFile, "service-account-signing-key-file", s.ServiceAccountSigningKeyFile, ""+
 		"Path to the file that contains the current private key of the service account token issuer. The issuer will sign issued ID tokens with this private key.")
+
+	fs.StringVar(&s.ServiceAccountSigningEndpoint, "service-account-signing-endpoint", s.ServiceAccountSigningEndpoint, ""+
+		"Path to socket where a external JWT signer is listening")
```

Comment:
document this requires the alpha feature gate enabled in the help text, and that it is mutually exclusive with `--service-account-signing-key-file` and `--service-account-key-file`

Reply:
in Options#Validate:
* enforce the flags are mutually exclusive
* reject a value in ServiceAccountSigningEndpoint if the feature gate is disabled
* maaaaybe validate the format of the flag value if provided (whatever minimal validation makes sure this is a good endpoint to pass to grpc init)

Reply:
Done